ndex2/client.py

```diff
@@ -965,8 +1362,36 @@ def grant_networks_to_group(self, groupid, networkids, permission="READ"):
 
     def get_user_by_username(self, username):
         """
+
         Gets the user id by user name
 
+        .. code-block:: python
+
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		print("Please enter your user name:")
+		name = input()
+
+		#this method returns the user information as a dict
+		user_aspect = my_ndex.get_user_by_username(name)
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client